### PR TITLE
CORE-16320 - Remove state-manager schema bootstrap configuration 

### PR DESCRIPTION
**Pull request by conal-smith**
**Description**
In several places we are using state manager schema from helm chart values.yaml but the worker itself cannot yet be configured to use a different schema.

**Solution**
Hard-code schema to STATE_MANAGER for now, so we can make this configurable in the future if necessary. Remove configurability of the schema in helm bootstrapping.

### CODE REVIEW

**Review by jujoramos — APPROVED** [review]
LGTM

**Comment by davidcurrie** [comment]
> so we can make this configurable in the future if necessary

I believe that this is already necessary in that we have environments that share a single database instance and rely on being able to override the schema for multi-tenancy.

@anton-subbotin / @Peter-Mitcham - can you confirm whether my statement above is true?

**Comment by anton-subbotin** [comment]
> > so we can make this configurable in the future if necessary
> 
> I believe that this is already necessary in that we have environments that share a single database instance and rely on being able to override the schema for multi-tenancy.
> 
> @anton-subbotin / @Peter-Mitcham - can you confirm whether my statement above is true?

@davidcurrie 
> we have environments that share a single database instance

that is true

> rely on being able to override the schema for multi-tenancy

not sure whether this might be required in the future, but previously in case of sharing a single Aurora RDS instance we didn't need to override the schema (in other words, we weren't overriding the affected parameter.

**Comment by anton-subbotin** [comment]
@davidcurrie but assuming this:
> but the worker itself cannot yet be configured to use a different schema.

do you think that we still can have a valid scenario for:
> share a single database instance and rely on being able to override the schema for multi-tenancy
?

**Comment by anton-subbotin** [comment]
@conalsmith-r3  not critical (as it shouldn't break anything), but would be nice to make a similar update in https://github.com/corda/corda5-enterprise/tree/release/ent/5.1/charts/corda-enterprise

**Comment by davidcurrie** [comment]
@anton-subbotin - well, the state manager DB is new. Do you know if any of the QA tests currently override the config/crypto/RBAC schemas to allow multiple tests to proceed in parallel using the same DB instance?

**Comment by anton-subbotin** [comment]
@davidcurrie 
>Do you know if any of the QA tests currently override the config/crypto/RBAC schemas to allow multiple tests to proceed in parallel using the same DB instance?

No, as far as I know. Do I understand it right that schema override would allow to share the same named _database entity_ (referred to as `$.Values.db.cluster.database`) ? If yes, then in the flow we've been using/orchestrating, the _database_ is unique per Corda deployment

**Comment by conal-smith** [comment]
> @davidcurrie
> 
> > Do you know if any of the QA tests currently override the config/crypto/RBAC schemas to allow multiple tests to proceed in parallel using the same DB instance?
> 
> No, as far as I know. Do I understand it right that schema override would allow to share the same named _database entity_ (referred to as `$.Values.db.cluster.database`) ? If yes, then in the flow we've been using/orchestrating, the _database_ is unique per Corda deployment

Schema override here would just change the schema inside $.Values.db.cluster.database.

If you deploy a separate cluster on the same RDS instance with a different $.Values.db.cluster.database, you'll have a separate state-manager bootstrapped in that database.

e.g. if two Corda helm installs override `$.Values.db.cluster.database` with "test-cluster-1" and "test-cluster-2" respectively, then the single RDS instance will have two databases:
- "test-cluster-1" 
  - "STATE_MANAGER"
  - "config"
  - "rbac"
- "test-cluster-2"
  - "STATE_MANAGER"
  - "config"
  - "rbac"

Soon, when we complete https://github.com/corda/corda-runtime-os/pull/4763, the state-manager will not be in the cluster db instance and the state-manager host and database will be configurable.

In summary, I think (and correct me if I'm wrong) we can go ahead with this change, schema here won't impact deployment of multiple clusters against the same RDS instance.

**Comment by davidcurrie** [comment]
Agreed - I thought we might be using a single database with different schema but it sounds like that is not the case. We should make sure that the docs indicate that anyone using the CLI directly needs to specify `STATE_MANAGER` as the schema.

**Review by davidcurrie — APPROVED** [review]

**Review by anton-subbotin — APPROVED** [review]
